CORRELATION_RULES_STYLE_GUIDE.md

```diff
@@ -0,0 +1,70 @@
+# panther-analysis Correlation Rule (CR) Style Guide and Best Practices
+
+This style guide highlights essential best practices for writing correlation rules. For a more detailed guide, visit [Correlation Rules](https://docs.panther.com/detections/correlation-rules) in the Panther documentation.
+
+## General advises
+
+- Transition names in rule IDs, and transition IDs should be all caps.
+    - e.g. `"GitHub Advanced Security Change NOT FOLLOWED BY repo archived"`
+- Boilerplate comments from the CR template in the UI should be removed before committing to PA.
+    - e.g. remove `# Create a list of rules to correlate`
+- Sequence, Group, and Transition IDs should have meaningful names.
+    - e.g. `- ID: GHASChange` instead of `- ID: TR.1`
+  - Check for Signal rules that already exist.
+    - e.g. we do not want duplicates of `AWS Console Login` rules for every CR that relies on that signal
+    - for the guide on Signal rules, please visit [How to create a rule that only produces signals](https://docs.panther.com/detections/correlation-rules/signals#signal-use-cases:~:text=Data%20Explorer.-,How%20to%20create%20a%20rule%20that%20only%20produces%20signals,-To%20create%20a)
+- Correlation rules go in `correlation_rules` directory, sub-rules and signals go in the appropriate logtype directory
+
+## Correlation rule fields
+
+### MinMatchCount
+
+`MinMatchCount: 1` is the default value, this can be omitted.
```

Comment:
in other sections within "Correlation rule fields," the body is formatted as bullets. I would suggest making this line a bullet, too

Reply:
suggested rewrite: If you are setting `MinMatchCount` to `1`, you can omit it completely, as `1` is the default value.